specificatie/genereervariant/openapi.yaml

```diff
@@ -311,9 +311,15 @@ components:
           maxItems: 130
           minItems: 1
           type: array
-          description: |
-            Hiermee kun je de inhoud van de resource naar behoefte aanpassen door een lijst van paden die verwijzen naar de gewenste velden op te nemen ([zie functionele specificaties 'fields' properties](https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/develop/features/fields.feature)).
-            De te gebruiken paden zijn beschreven in [fields-Persoon.csv](https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/develop/features/fields-Persoon.csv) (voor gebruik fields bij raadplegen) en [fields-PersoonBeperkt.csv](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/develop/features/fields-PersoonBeperkt.csv) (voor gebruik fields bij zoeken) waarbij in de eerste kolom het fields-pad staat en in de tweede kolom het volledige pad naar het gewenste veld.
+          description: "Hiermee kun je de velden opgeven die je wenst te ontvangen.\n\
+            De te gebruiken paden zijn beschreven in [fields-Persoon.csv](https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/develop/features/fields-Persoon.csv)\
+            \ \n(voor gebruik fields bij raadplegen) en [fields-PersoonBeperkt.csv](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/develop/features/fields-PersoonBeperkt.csv)\
+            \ \n(voor gebruik fields bij zoeken).\nVelden die -indien van toepassing-\
+            \ automatisch worden geleverd (inOnderzoek, geheimhoudingPersoonsgegevens,\
+            \ opschortingBijhouding, rni en verificatie) hoeven en mogen niet worden\
+            \ opgegeven in fields.\nBij opgave van een pad dat verwijst naar een niet-bestaand\
+            \ veld of naar een automatisch geleverd veld wordt een 400 Bad Request\
+            \ teruggegeven.\n"
```

Comment:
Velden die automatisch worden geleverd (inOnderzoek, geheimhoudingPersoonsgegevens, opschortingBijhouding, rni en verificatie) mag je niet opgeven in fields.\n Opgave van een pad dat verwijst naar een niet-bestaand veld of naar een automatisch geleverd veld leidt tot een 400 Bad Request.\n",